lib/__tests__/standalone-cache.test.js

```diff
@@ -36,134 +39,108 @@ function getConfig() {
 describe('standalone cache', () => {
 	const expectedCacheFilePath = path.join(cwd, '.stylelintcache');
 
-	beforeEach(() => {
+	beforeEach(async () => {
 		// Initial run to warm up the cache
-		return standalone(getConfig());
+		await standalone(getConfig());
 	});
 
-	afterEach(() => {
+	afterEach(async () => {
 		// Clean up after each test case
-		return Promise.all([
-			fs.unlink(expectedCacheFilePath).catch(() => {
-				// fs.unlink() throws an error if file doesn't exist and it's ok. We just
-				// want to make sure it's not there for next test.
-			}),
-			fs.unlink(newFileDest).catch(() => {
-				// fs.unlink() throws an error if file doesn't exist and it's ok. We just
-				// want to make sure it's not there for next test.
-			}),
-		]);
+		await removeFile(expectedCacheFilePath);
+		await removeFile(newFileDest);
 	});
 
-	it('cache file is created at $CWD/.stylelintcache', () => {
+	it('cache file is created at $CWD/.stylelintcache', async () => {
 		// Ensure cache file exists
-		return fileExists(expectedCacheFilePath).then((isFileExist) => {
-			expect(Boolean(isFileExist)).toBe(true);
+		expect(existsSync(expectedCacheFilePath)).toBe(true);
 
-			const fileCache = fCache.createFromFile(expectedCacheFilePath);
-			const { cache } = fileCache;
+		const { cache } = fCache.createFromFile(expectedCacheFilePath);
 
-			// Ensure cache file contains only linted css file
-			expect(typeof cache.getKey(validFile) === 'object').toBe(true);
-			expect(typeof cache.getKey(newFileDest) === 'undefined').toBe(true);
-		});
+		// Ensure cache file contains only linted css file
+		expect(typeof cache.getKey(validFile)).toBe('object');
+		expect(cache.getKey(validFile)).toBeTruthy();
```

Comment:
[note] I've added the expectation `.toBeTruthy()` because of `typeof null === 'object'`. But it may be a bit ugly and is not DRY. 🤔